cli/tests/pcluster/cli/test_build_image.py

```diff
@@ -89,23 +89,34 @@ def test_error(self, mocker, test_datadir):
             run(command)
         assert_that(exc_info.value.data).is_equal_to(api_response[0])
 
+    @staticmethod
+    def run_build_image_command(test_datadir):
+        run(
+            [
+                "build-image",
+                "--region",
+                "eu-west-1",
+                "--image-configuration",
+                str(test_datadir / "config.yaml"),
+                "--image-id",
+                "image-id",
+            ]
+        )
+
     def test_no_nodejs_error(self, mocker, test_datadir):
         """Test expected message is printed out if nodejs is not installed."""
         mocker.patch("pcluster.api.util.shutil.which", return_value=None)
         with pytest.raises(APIOperationException) as exc_info:
-            run(
-                [
-                    "build-image",
-                    "--region",
-                    "eu-west-1",
-                    "--image-configuration",
-                    str(test_datadir / "config.yaml"),
-                    "--image-id",
-                    "image-id",
-                ]
-            )
+            self.run_build_image_command(test_datadir)
         assert_that(exc_info.value.data.get("message")).matches("Node.js is required")
 
+    def test_nodejs_wrong_version_error(self, mocker, test_datadir):
```

Comment:
I think we should move these tests from test_build_image to test_api_util. py to reflect the module they are defined. Then they are not strictly related to build_image

Reply:
adde standalone unit test as well